composer.json

```diff
@@ -22,7 +22,7 @@
         "php": ">=7.3.0",
         "ext-curl": "*",
         "ext-json": "*",
-        "mnapoli/silly": "^1.7",
+        "symfony/console": "^3.0|^4.0|^5.0|^6.0",
```

Comment:
Is it necessary to support old and unmaintained versions? `^4.4|^5.0|^6.0` should be enough.

Reply:
I just copied the requirement from Silly, and then added `^6.0`. I'd suggest to keep support for the old releases for now, and to think about it together with the other Symfony requirements in another issue/PR.